### PR TITLE
scan stdin by byte instead of line

### LINKED ISSUE #40: Long plan output gives "cannot parse plan result"
## WHAT

We have a very long Terraform Plan output with a lot of inline JSON that cuauses the following error:

```
$ terraform plan terraform-prod.plan | tfnotify plan 

The plan command received a saved plan file as input. This command
will output the saved plan. This will not modify the already-existing
plan. If you wish to generate a new plan, please pass in a configuration
directory as an argument.



An execution plan has been generated and is shown below.
Resource actions are indicated with the following symbols:
  + create
  ~ update in-place

Terraform will perform the following actions:

  ~ module.apps.helm_release.prometheus-operator
      values.0:                     "global:\n  rbac:\n    enabled: true\ncommonLabels:\n  prometheus: default\ndefaultRules:\n  labels:\n    alertmanager: default\n  rules:\n    alertmanager: true\n    etcd: false\n    general: true\n    k8s: true\n    kubeApiserver: false\n    kubePrometheusNodeAlerting: true\n    kubePrometheusNodeRecording: true\n    kubeScheduler: false\n
…
(removed looooooooong output here)
…
eShift\": null,\n      \"title\": \"NATS Pending: Account Created\",\n      \"tooltip\": {\n        \"shared\": true,\n        \"sort\": 0,\n        \"value_type\": \"individual\"\n      },\n      \"type\": \"graph\",\n      \"xaxis\": {\n        \"buckets\": null,\n        \"mode\": \"time\",\n        \"name\": null,\n        \"show\": true,\n        \"values\": []\n      },\n      \"yaxes\": [\n        {\n          \"format\": \"short\",\n          \"label\": null,\n          \"logBase\": 1,\n          \"max\": null,\n          \"min\": null,\n          \"show\": true\n        },\n        {\n          \"format\": \"short\",\n          \"label\": null,\n          \"logBase\": 1,\n          \"max\": null,\n          \"min\": null,\n          \"show\": true\n        }\n      ],\n      \"yaxis\": {\n        \"align\": false,\n        \"alignLevel\": null\n      }\n    }\n  ],\n  \"refresh\": \"5s\",\n  \"schemaVersion\": 18,\n  \"style\": \"dark\",\n  \"tags\": [],\n  \"templating\": {\n    \"list\": []\n  },\n  \"time\": {\n    \"from\": \"now-1h\",\n    \"to\": \"now\"\n  },\n  \"timepicker\": {\n    \"refresh_intervals\": [\n      \"5s\",\n      \"10s\",\n      \"30s\",\n      \"1m\",\n      \"5m\",\n      \"15m\",\n      \"30m\",\n      \"1h\",\n      \"2h\",\n      \"1d\"\n    ],\n    \"time_options\": [\n      \"5m\",\n      \"15m\",\n      \"1h\",\n      \"6h\",\n      \"12h\",\n      \"24h\",\n      \"2d\",\n      \"7d\",\n      \"30d\"\n    ]\n  },\n  \"timezone\": \"\",\n  \"title\": \"Onboarding\",\n  \"uid\": \"0_R1z5MWk\",\n  \"version\": 6\n}"
cannot parse plan result
Exited with code 1
```

From the looks of it `tfnotify` just gives up half way through and can not find what it is looking for in order to classify this as a valid plan.

```
$ terraform plan terraform-test.plan | wc
      86   20863  325582
```

As you can see the plan is quit big, this would definitly hit the GitHub comment limit #34, but this should still not fail in `tfnotify`, don't you think?

This is the latest version of tfnotify and the latest v0.11 version of Terraform.

### PR DESCRIPTION
**Pull request by timothegenzmer**
## WHAT

Scan stdin by byte instead of line

## WHY

this fixes #40

### CODE REVIEW

**Comment by timothegenzmer** [comment]
Could someone review this?
Is there any additional information needed?

**Comment by timothegenzmer** [comment]
hey thanks for the reply.

When I run `go fmt` only `ci.go` was formatted.
Could you be more specific what is not formatted correctly?

**Comment by dtan4** [comment]
Since `ci.go` is formatted correctly in the current master branch, you don't need to run `go fmt` here.

Instead, could you merge the current master branch into this branch? Thanks!

File: tee.go [hunk 1]
```diff
@@ -3,10 +3,8 @@ package main
 import (
 	"bufio"
 	"bytes"
-	"fmt"
-	"io"
-
 	"github.com/mattn/go-colorable"
+	"io"
```
Comment (b4b4r07) [inline tee.go]: I mean this!

this should be like this:

```go
	"io"	

	"github.com/mattn/go-colorable"
```

Comment (timothegenzmer) [reply tee.go]: done :)

**Comment by dtan4** [comment]
@timothegenzmer 

Released as [v0.5.3](https://github.com/mercari/tfnotify/releases/tag/v0.5.3). Thank you for your contribution!

**Pull request merged**